### PR TITLE
[Tabs] Re-enable new ink (reverts #2665)

### PR DESCRIPTION
**Pull request by jverkoey**
This reverts commit 83994481a936a47d4bec4381fa34f711c47ba486.

### CODE REVIEW

**Comment by jverkoey** [comment]
I believe I have a strategy that will allow us to resolve the internal client team regressions in the next release.

**Review by randallli — APPROVED** [review]

File: components/Tabs/src/private/MDCItemBarCell.m [hunk 1]
```diff
@@ -562,6 +562,8 @@ - (void)updateInk {
   MDCInkView *inkView = _inkTouchController.defaultInkView;
   inkView.inkColor = _style.inkColor;
   inkView.inkStyle = _style.inkStyle;
+  inkView.usesLegacyInkRipple = NO;
+  inkView.clipsToBounds = inkView.inkStyle == MDCInkStyleBounded ? YES : NO;
```
Comment (romoore) [inline components/Tabs/src/private/MDCItemBarCell.m]: Nit: Please use parentheses here just to improve legibility.

**Review by romoore — APPROVED** [review]

**Comment by romoore** [comment]
Unrelated failing unit test in Travis:
```
Test CFailing tests:
	-[SnackbarManagerTests testMessagesResumedWhenTokenIsDeallocated]
```

**Comment by romoore** [comment]
Other failure is caused by MDCOverlayObserver being retained. Filed #2685 to track.

**Pull request merged**